src/technologies/a.json

```diff
@@ -1537,16 +1537,34 @@
     "scripts": "aem-(?:GridColumn|apps/)",
     "website": "https://www.adobe.com/marketing/experience-manager.html"
   },
-  "Adobe Experience Manager Franklin": {
+  "Adobe Experience Manager with Edge Delivery Services preview": {
     "cats": [
       1
     ],
     "cpe": "cpe:2.3:a:adobe:experience_manager:*:*:*:*:*:*:*:*",
-    "description": "Adobe Experience Manager Franklin, also known as Project Helix or Composability, is a new way to publish AEM pages using Google Drive or Microsoft Office via Sharepoint. Instead of components, Franklin uses blocks to build pages. Blocks are pieces of a document that will be transformed into web page content.",
-    "icon": "Adobe Experience Manager Franklin.svg",
-    "excludes": "Adobe Experience Manager",
-    "scriptSrc": "^.+/scripts/lib-franklin\\.js$",
-    "website": "https://www.hlx.live"
+    "description": "Edge Delivery Services in Adobe Experience Manager Sites as a Cloud Service (Preview)",
+    "icon": "Adobe Experience Platform.svg",
+    "saas": "true",
+    "url": "https.+\\.(aem|hlx)\\.(page|live)",
+    "website": "https://www.aem.live"
+  },
+  "Adobe Experience Manager Edge Delivery Services": {
+    "cats": [
+      1
+    ],
+    "cpe": "cpe:2.3:a:adobe:experience_manager:*:*:*:*:*:*:*:*",
+    "description": "Edge Delivery Services in Adobe Experience Manager Sites as a Cloud Service, also know as Helix, Franklin, Next-Generation Composability is a fast and lightweight content management system from Adobe.",
+    "icon": "Adobe Experience Platform.svg",
+    "saas": "true",
+    "excludes": [
+      "Adobe Experience Manager",
```

Comment:
Why does this exclude "Adobe Experience Manager" when it's in the name? Presumable because this is a separate product?

Also why does "Adobe Experience Manager with Edge Delivery Services preview" not exclude "Adobe Experience Manager"?

Reply:
The Adobe Experience Manager rule you had already in place was aimed at AEM as a cloud service with publish tier.

Edge Delivery Services is the new delivery tier that we launched last year, and we would recommend to have this one tracked separately.

The Preview service is only applicable to Edge Delivery services, hence I only excluded it in the Edge Delivery Services rule.
Actually, the Preview service should not even be tracked at all (non-prod environments).

Reply:
Ah OK. I guess we could rename "Adobe Experience Manager" to make it clearer, but then lose ability to see histroy of that so probably fine.

> Actually, the Preview service should not even be tracked at all (non-prod environments).

Suggesting removing that for reasons given in other comment.

Resolving this.